dev/integration_tests/codegen/android/.project

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe revert this and clean it up in a separate PR, potentially by adding something to gitignore?

Reply:
Going to file a separate PR, this was unintentional.  Cleaned it up for now here.